internal/services/formatters/hcl/checkov/formatter.go

```diff
@@ -15,6 +15,7 @@
 package checkov
 
 import (
+	"bytes"
 	"encoding/json"
 
 	"github.com/pborman/ansi"
```

Comment:
is possible remove this package ? last commit was 2016 is not good usage

Reply:
This code was not added on this pr, so I think that we need to deal with this in other pr.